presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1642,7 +1642,9 @@ public HiveOutputTableHandle beginCreateTable(ConnectorSession session, Connecto
                     handle.getTableName(),
                     partitionObjectBuilder.buildPartitionObject(session, table, update, prestoVersion, partitionParameters),
                     update.getWritePath(),
-                    partitionStatistics);
+                    partitionStatistics,
+                    true,
+                    writeInfo.getTargetPath());
```

Comment:
This is incorrect, as it has to be set to the table location, not to the partition location

Reply:
`writeInfo.getTargetPath()` was being used to set table location,  will change to `table.getStorage().getLocation()` to make it clear.